it/connectors/NrsConnectorISpec.scala

```diff
@@ -83,226 +72,121 @@ class NrsConnectorISpec extends ComponentSpecBase {
 
   private val CHECKSUM_FAILED = 419
 
-  val mockConfig: MockAppConfig = app.injector.instanceOf[MockAppConfig]
   val httpClient: HttpClient = app.injector.instanceOf[HttpClient]
-  val connector: NrsConnector = new NrsConnector(httpClient, mockConfig)
+  val connector: NrsConnector = new NrsConnector(httpClient, mockAppConfig)
 
   "nrsReceiptSubmission is called and the feature switch is off" should {
+    
```

Comment:
Isn't this feature being enabled before all tests? So for each test it needs to be disabled